### ISSUE TITLE
Back button not working

### ISSUE THREAD

**Issue by yaroslavkulinich** [mollyIV/ChatHeads#2]
Replace `WindowManager.LayoutParams.FLAG_NOT_TOUCH_MODAL` with `WindowManager.LayoutParams.FLAG_NOT_TOUCH_MODAL| WindowManager.LayoutParams.FLAG_NOT_FOCUSABLE` when configuring LayoutParams in `addToWindowManager()` method to allow users to normally use back button.

**Comment by mollyIV** [comment]
Hi @mtwain 
thank you for requesting a new feature. Actually it's not as simple as replacing the flags in a service. You need to run an extra service with transparent overlay with particular flags set. I'll be working on that soon :)

**Comment by rubinbasha** [comment]
I used your solution also for creating a chat heads like view and i also had issues with the back button and also while the view was shown than i could not show dialogs when long clicking other applications(like dialogs that pop up after long clicking a contact in the contacts app in my LG G4 device.) The solution in my case was to just use `WindowManager.LayoutParams.FLAG_NOT_FOCUSABLE` . @mollyIV  I am curious what might be the issue with this approach, since you say in your comment that it is not as simple. What do you mean by that?

**Comment by djdance** [comment]
Agree, WindowManager.LayoutParams.FLAG_NOT_FOCUSABLE allows me to access checkbox after running, overwise not.

Steps to reproduce:
1. check service
2. press Home
3. call our SETTINGS again from system bar
4. can't click anymore!
